### DISCUSSION TITLE
Is there a way to display a formatted date from NextUI's DatePicker and DateInput?

### DISCUSSION THREAD

**Discussion by ZaidAli3642** [heroui-inc/heroui#3836] [Q&A]
I'm using the NextUI library version 2.3.6. I'd like to use the [DatePicker](https://nextui.org/docs/components/date-picker) or the [DateInput](https://nextui.org/docs/components/date-input).

However, the default format displayed is mm/dd/yyyy; I need to display the date in the following format: dd/mm/yyyy.

Is there any way of using a format other than the default within a DatePicker or a DateInput?

**Comment by wingkwong** [comment]
You can set the locale in the provider. see https://nextui.org/docs/api-references/nextui-provider#props
